src/msalexample-ios-macos/Shared/MSALAuthentication.swift

```diff
@@ -0,0 +1,56 @@
+import MSAL
+
+class MSALAuthentication {
+    private static let kClientId = "APPLICATION_(CLIENT)_ID"
+    private static let kTenantId = "TENANT_ID"
+    private static let kAuthority = "https://login.microsoftonline.com/"
+    
+    private static var kApplication: MSALPublicClientApplication?
+    
+    public static func Signin(_ webviewParameters: MSALWebviewParameters, completion: @escaping (MSALAccount?, _ accessToken: String?, Error?) -> Void) {
+        let authority = try? MSALB2CAuthority(url: URL(string: "\(kAuthority)\(kTenantId)")!)
+        let config = MSALPublicClientApplicationConfig(clientId: kClientId, redirectUri: nil, authority: authority)
+        
+        kApplication = try? MSALPublicClientApplication(configuration: config)
+        
+        let webviewParameters = webviewParameters
+        let interactiveParameters = MSALInteractiveTokenParameters(scopes: ["user.read"], webviewParameters: webviewParameters)
```

Comment:
Looks like you're always taking the user down the interactive path.  Wouldn't we start with the `MSALSilentTokenParameters` -> `acquireTokenSilent`, with a check for the `interactionRequired` error?  Especially here in an mobile/desktop app that would likely be taking heavy advantage of the cached tokens.

Reply:
thanks for feedback @ckittel, yeah I also feel that it'd be great teachable moment but there is a technical impediment for such pattern to be implement in `macOS`. Therefore, please let me recommend to keep this tutorial as simple as possible, and add a comment as we've agreed from other tutorials. Our content devs can always decide to link to a full fleshed QuickStart and/or RI where such pattern are already implemented. 

partially done | addressed from [7af7f2](https://github.com/Azure-Samples/ms-identity-docs-code-swift/pull/1/commits/7af7f2cf904fa200317a9dc2fe95f793afe3a7f9)

Reply:
The comment covers the "teachable moment" part of this well, thanks!

Reply:
I'm cool with a teachable moment, but we (content devs, at least) need to see how to do it the _right_ way before we can show both in a teachable moment. This seems like the only sample thus far where we're showing an anti-pattern as default, and this is a case where I think more code would be OK so we don't lead with (and include only) the anti-pattern.

Right now, I'm kinda getting this feeling:

![image](https://user-images.githubusercontent.com/13394541/157997409-db0f1c04-5ea3-4b99-9844-0b6038e2e367.png)